### PR TITLE
Remove onnxruntime team from CODEOWNERS

### PR DESCRIPTION
**Pull request by garymm**
There are currently 98 members in the team. Requesting review from
all of them for every PR is too noisy. Individuals can subscribe to the
repo if they want to receive notifications for every PR.

### CODE REVIEW

**Review by codemzs — APPROVED** [review]

**Review by pranavsharma — DISMISSED** [review]

**Comment by pranavsharma** [comment]
@garymm - will github still **require** a review before merging PRs if the files included in the PRs don't match the whitelisted paths mentioned in the CODEOWNERS file? If not, we can't remove the onnxruntime line. 
Like other repos, we've the following configured for the master branch.

image

**Review by thiagocrepaldi — APPROVED** [review]

**Comment by garymm** [comment]
> @garymm - will github still **require** a review before merging PRs if the files included in the PRs don't match the whitelisted paths mentioned in the CODEOWNERS file?

I believe you can configure GH to require reviews for all PRs by selecting "Require pull request reviews before merging" in "[Branch protection rules](https://docs.github.com/en/github/administering-a-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule)".

@pranavsharma I'll wait for your approval before merging this.

**Review by pranavsharma — APPROVED** [review]

**Comment by pranavsharma** [comment]
> > @garymm - will github still **require** a review before merging PRs if the files included in the PRs don't match the whitelisted paths mentioned in the CODEOWNERS file?
> 
> I believe you can configure GH to require reviews for all PRs by selecting "Require pull request reviews before merging" in "[Branch protection rules](https://docs.github.com/en/github/administering-a-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule)".

This is already enabled.

**Review by natke — APPROVED** [review]

**Pull request merged**
